src/logic/calculate.js

```diff
@@ -0,0 +1,20 @@
+import operate from './operate';
+/* eslint-disable no-param-reassign */
+
+const calculate = (calculator, buttonName) => {
+  let { total, next } = calculator;
```

Comment:
`calculator` object should have 03 properties [ [screenshot](https://gitlab.com/microverse/guides/projects/requirements_screenshots/raw/master/images/react_reducx/calculator/milestone4/totalNextOperate.png) ]
- total
- next &&
- operation